Examples/Movies/android/app/src/main/java/com/facebook/react/movies/MoviesActivity.java

```diff
@@ -35,7 +36,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     mReactInstanceManager = ReactInstanceManager.builder()
         .setApplication(getApplication())
-        .setBundleAssetName("MoviesApp.android.bundle")
+        .setJSBundleLoader(JSBundleLoader.createAssetLoader("MoviesApp.android.bundle"))
```

Comment:
Were there objections to just going with setBundleFile(file://android_assets/MoviesApp.android.bundle)? (or assets:// if that doesn't work).